packages/widgetbook/test/src/addons/test_scale_factor/text_scale_test_utilities.dart

```diff
@@ -0,0 +1,76 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:provider/provider.dart';
+import 'package:widgetbook/src/addons/addon.dart';
+import 'package:widgetbook/src/addons/text_scale_addon/text_scale_selection_provider.dart';
+
+import '../utils/addons.dart';
+import '../utils/custom_app_theme.dart';
+import '../utils/theme_wrapper.dart';
+
+TextScaleSettingProvider getTextScaleFactorSettingProvider(
```

Comment:
We could have a general method for this as well, so we can re-use it in all tests